src/v2/Apps/Order/Routes/Payment/PaymentContent.tsx

```diff
@@ -74,7 +91,13 @@ export const PaymentContent: FC<Props> = props => {
           {paymentMethod && (
             <Clickable
               textDecoration="underline"
-              onClick={() => setStepOneComplete(false)}
+              onClick={() => {
+                trackClickedPaymentMethod(
+                  "click change payment method",
```

Comment:
With the [new design](https://www.figma.com/file/rhcgsn0cMrogxN0TXu5l6T/TX-Bank-transfer?node-id=461%3A7555), the "change" and "save and continue" buttons will no longer be there, and users will be able to change payment methods by just selecting the other radio button, so that will change things here. Other than that looks great so far!